docs/developer/testing/TESTING.md

```diff
@@ -1,61 +1,87 @@
 <h1 align="center">E2E Testing</h1>
 
-## Prerequisites:
+## Prerequisites
 
 ### Install Ginkgo
+
+<!-- Can we add this as pre run for make test-e2e? -->
 ```bash
-$ go get -u github.com/onsi/ginkgo/v2/ginkgo
+go install -v github.com/onsi/ginkgo/v2/ginkgo
 ```
 
 ### Setup backup storage configuration
+
 To get started, the test suite expects 2 files to use as configuration for
 Velero's backup storage. One file that contains your credentials, and another
-that contains additional configuration options (for now the name of the
+that contains additional configuration options (for now, the name of the
 bucket).
 
 The default test suite expects these files in `/var/run/oadp-credentials`, but
-can be overridden with the environment variables `OADP_AWS_CRED_FILE` and
-`OADP_S3_BUCKET`.
+can be overridden with the environment variables `OADP_CRED_FILE` and
+`OADP_BUCKET_FILE`.
 
 To get started, create these 2 files:
-`OADP_AWS_CRED_FILE`:
+`OADP_CRED_FILE`:
 ```
 [<INSERT_PROFILE_NAME>]
 aws_access_key_id=<access_key>
 aws_secret_access_key=<secret_key>
 ```
+If you use one profile name different from `default`, also change `BSL_AWS_PROFILE`.
 
-`OADP_S3_BUCKET`:
-```json
-{
-  "velero-bucket-name": <bucket>
-}
+<!-- change to env var? -->
+`OADP_BUCKET_FILE`:
+```
+<bucket_name>
 ```
 
 ## Run all e2e tests
+
+<!-- How this is run in CI? Can we add this as pre run for make test-e2e? -->
+TODO set needed variables
```

Comment:
Instead of changing the file, `export name=value` works as well, right?

Reply:
right